README.md

```diff
@@ -1 +1,60 @@
-# bookstore
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# Bookstore CMS
+The Bookstore is a website that helps you;
+- `Display a list of books.`
+- `Add a book.`
+- `Remove a selected book.`
+
+
+![screenshot](./pic.png)
+
+
+# Built with
+- React.js
+- HTML
+- CSS
+# Other tools
+- Babel compiler
+# Getting Started with Create React App
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## installation
+In order to run on your local machine, type in;
+
```

Comment:
- [ ] please make sure to use[ the template](https://github.com/microverseinc/readme-template) for the readme file provided by Microverse, including the links (for example to the MIT license). keep in mind that all sections of the template are mandatory except for the "Live Demo", "Run Tests" and "Deployment" sections, these are only mandatory if the specific project you're working on requires it.